src/main/java/seedu/duke/ui/Ui.java

```diff
@@ -41,21 +41,23 @@
         "Hooray! You're set to use the circuit builder of \n" + LOGO + "Enjoy! :)\n \ttype 'exit' to exit this mode."
             + "\n"};
 
-    public static final String[] BOOLEAN_INSTRUCTIONS = {"Lets get started! First chose a template you want to work "
+    public static final String[] BOOLEAN_INSTRUCTIONS = {"Let's get started! First chose a template you want to work "
```

Comment:
Maybe consider using `\t` similar to what @sevenseasofbri did for `CIRCUIT_INSTRUCTIONS`? To be consistent, `\n` can also be replaced with `System.lineSeparator()`.